services/motion/builtin/builtin_test.go

```diff
@@ -236,8 +236,8 @@ func TestMoveOnMap(t *testing.T) {
 		slam.Named("test_slam"),
 		nil,
 	)
-	test.That(t, err, test.ShouldBeError, builtin.ErrNotImplemented)
-	test.That(t, success, test.ShouldBeFalse)
+	test.That(t, err, test.ShouldBeNil)
```

Comment:
[q]: isnt it the case that `*fake.Base` fails the type assertion of `base.KinematicWrappable`, i.e. where you do:
```
	// create a KinematicBase from the componentName
	component, ok := ms.components[componentName]
	if !ok {
		return false, resource.DependencyNotFoundError(componentName)
	}
	kw, ok := component.(base.KinematicWrappable)
	if !ok {
		return false, fmt.Errorf("cannot move component of type %T because it is not a KinematicWrappable Base", component)
	}
```